lib/WebpackConfig.js

```diff
@@ -708,6 +713,17 @@ class WebpackConfig {
         this.handlebarsConfigurationCallback = callback;
     }
 
+    enableBrowserSync(paths, browserSyncOptionsCallback = () => {}) {
+        this.useBrowserSync = true;
+        this.browserSyncOptions.paths = paths;
```

Comment:
Should we also check if `paths` is an `Array` here? We could also allow `string` and put it into an array to allow enabling BrowserSync on a single path.